Mathlib/Analysis/NormedSpace/Alternating/Basic.lean

```diff
@@ -0,0 +1,601 @@
+  __ := compContinuousAlternatingMapL 𝕜 E F G g.toContinuousLinearMap
+  continuous_toFun :=
+    (compContinuousAlternatingMapL 𝕜 E F G g.toContinuousLinearMap).continuous
+  continuous_invFun :=
+    (compContinuousAlternatingMapL 𝕜 E G F g.symm.toContinuousLinearMap).continuous
+
+@[simp]
+theorem _root_.ContinuousLinearEquiv.continuousAlternatingMapCongrRight_symm (g : F ≃L[𝕜] G) :
+    (g.continuousAlternatingMapCongrRight (ι := ι) (E := E)).symm =
+      g.symm.continuousAlternatingMapCongrRight :=
+  rfl
+
+/-- Flip arguments in `f : F →L[𝕜] E [⋀^ι]→L[𝕜] G` to get `⋀^ι⟮𝕜; E; F →L[𝕜] G⟯` -/
+@[simps! apply_apply]
+def flipAlternating (f : F →L[𝕜] (E [⋀^ι]→L[𝕜] G)) : E [⋀^ι]→L[𝕜] (F →L[𝕜] G) where
+  toContinuousMultilinearMap :=
+    ((ContinuousAlternatingMap.toContinuousMultilinearMapCLM 𝕜).comp f).flipMultilinear
+  map_eq_zero_of_eq' v i j hv hne := by ext x; simp [(f x).map_eq_zero_of_eq v hv hne]
+
+end ContinuousLinearMap
+
+theorem LinearIsometry.norm_compContinuousAlternatingMap (g : F →ₗᵢ[𝕜] G) (f : E [⋀^ι]→L[𝕜] F) :
+    ‖g.toContinuousLinearMap.compContinuousAlternatingMap f‖ = ‖f‖ :=
+  g.norm_compContinuousMultilinearMap f.1
+
+open ContinuousAlternatingMap
+
+section
+
+theorem ContinuousAlternatingMap.norm_compContinuousLinearMap_le (f : F [⋀^ι]→L[𝕜] G)
+    (g : E →L[𝕜] F) : ‖f.compContinuousLinearMap g‖ ≤ ‖f‖ * (‖g‖ ^ Fintype.card ι) :=
+  (f.1.norm_compContinuousLinearMap_le _).trans_eq <| by simp
+
+/-- Composition of a continuous alternating map and a continuous linear map
+as a bundled continuous linear map. -/
+def ContinuousAlternatingMap.compContinuousLinearMapCLM (f : E →L[𝕜] F) :
+    (F [⋀^ι]→L[𝕜] G) →L[𝕜] (E [⋀^ι]→L[𝕜] G) :=
+  LinearMap.mkContinuous
+    (ContinuousAlternatingMap.compContinuousLinearMapₗ f) (‖f‖ ^ Fintype.card ι) fun g ↦
+      (g.norm_compContinuousLinearMap_le f).trans_eq (mul_comm _ _)
+
+/-- Given a continuous linear isomorphism between the domains,
+generate a continuous linear isomorphism between the spaces of continuous alternating maps. -/
+@[simps apply]
+def ContinuousLinearEquiv.continuousAlternatingMapCongrLeft (f : E ≃L[𝕜] F) :
+    E [⋀^ι]→L[𝕜] G ≃L[𝕜] (F [⋀^ι]→L[𝕜] G) where
+  __ := f.continuousAlternatingMapCongrLeftEquiv
+  __ := ContinuousAlternatingMap.compContinuousLinearMapCLM (f.symm : F →L[𝕜] E)
+  toFun g := g.compContinuousLinearMap (f.symm : F →L[𝕜] E)
+  continuous_invFun :=
+    (ContinuousAlternatingMap.compContinuousLinearMapCLM (f : E →L[𝕜] F)).cont
+  continuous_toFun :=
+    (ContinuousAlternatingMap.compContinuousLinearMapCLM (f.symm : F →L[𝕜] E)).cont
+
+variable
+  {E' : Type*} [NormedAddCommGroup E'] [NormedSpace 𝕜 E']
+  {F' : Type*} [NormedAddCommGroup F'] [NormedSpace 𝕜 F']
+
+/-- Continuous linear equivalences between the domains and the codomains
+generate a continuous linear equivalence between the spaces of continuous alternating maps. -/
+@[simps! apply]
+def ContinuousLinearEquiv.continuousAlternatingMapCongr (e : E ≃L[𝕜] E') (e' : F ≃L[𝕜] F') :
```

Comment:
I'd maybe argue that these "combine both congrs for convenience" definitions should be `abbrev`s, so that you can access lemmas about the individual parts more easily.

If what you've done is consistent with what we have for multilinear maps, then feel free to postpone this comment to an (optional) future cleanup.